Resources/ServerInfo/Guidebook/Antagonist/Revolutionaries.xml

```diff
@@ -0,0 +1,54 @@
+<Document>
+  # Revolutionaries
+
+  - Revolutionaries are antagonists that are sponsored by the Syndicate to take over the station.
+
+  ## Head Revolutionaries
+  
+<Box>
+  <GuideEntityEmbed Entity="Flash" Caption="Flash"/>
+  <GuideEntityEmbed Entity="ClothingEyesGlassesSunglasses" Caption="Sunglasses"/>
+</Box>
+
+  [color=#5e9cff]Head Revolutionaries[/color] are chosen at the start of the shift and are tasked with taking over the station by killing or exiling all of the Command staff. To aid you in this task you will be given a [color=#a4885c]Flash[/color] and a pair of [color=#a4885c]Sunglasses[/color].
```

Comment:
There shouldn't be a sudden shift to the second person, i.e. instead of "to aid you" maybe "Head revolutionaries are given..."